CONTRIBUTING.md

```diff
@@ -54,6 +56,37 @@ Please explain why someone else should merge your proposed change:
 * Does my PR make multiple changes and therefore require multiple commits?
 Ideally, each individual change correlates to an individual commit.
 
+## Capitalization
+
+All headings are sentence case.
+We capitalize the following terms:
+
+* Ansible
+* Candlepin
+* Chef
+* Content View
```

Comment:
Interesting that "Content View" is consistently capitalized in the documentation but lower case in the [UI](https://access.redhat.com/documentation/en-us/red_hat_satellite/6.14/html-single/managing_content/index#Creating_a_Composite_Content_View_content-management).

Reply:
Yeah, I am fairly certain that it was supposed to be lowercase to begin with. However, since currently all guides use the uppercase version, I propose to keep this here for now and then remove it in the PR changing CV to lowercase. Or should we say now that it shouldn't be uppercase to not confuse writers later? But then there might be confusion as to why is CV currently uppercase everywhere.

Reply:
https://github.com/theforeman/foreman-documentation/issues/2747